docs/Integrating Your HMI/Vehicle Data/index.md

```diff
@@ -0,0 +1,169 @@
+# Vehicle Data
+
+Vehicle data can be exposed to app developers by creating a `VehicleInfo` component within your HMI. The primary RPCs used by this component are:
```

Comment:
I would mention that it is required to send a MB.RegisterComponent message for the VehicleInfo interface, as well as needing to handle the IsReady request rom Core.

Reply:
Looks like most of this information is available in the HMI Getting Started guide, so I can link out to there.